aptos_sdk/async_client.py

```diff
@@ -30,12 +30,21 @@
 class ClientConfig:
     """Common configuration for clients, particularly for submitting transactions"""
 
-    expiration_ttl: int = 600
-    gas_unit_price: int = 100
-    max_gas_amount: int = 100_000
-    transaction_wait_in_seconds: int = 20
-    http2: bool = False
-    api_key: Optional[str] = None
+    def __init__(
```

Comment:
I think this custom init method could be avoided if you used a dataclass instead: https://docs.python.org/3/library/dataclasses.html.

Reply:
yeah. At the first time I wanted to use `dataclass` too,  but `dataclass` was added in Python 3.7 and I'm not sure the minimum version we wanted to support, so...😂

Reply:
Oh, updated to use `dataclass`
just noticed the minimum version has been set https://github.com/aptos-labs/aptos-python-sdk/blob/main/pyproject.toml#L20 🤣